SU-SWS/stanford_basic
# D8CORE-1396: increased the z-index on the header

# READY FOR REVIEW

# Summary
- Increased the z-index so the mobile hamburger menu lays over the card on the homepage instead of under it

# Review By (Date)
- 2/21

# Urgency
- med

# Steps to Test

1. Pull in this branch.
2. Build and look at the homepage in a mobile tablet size.
3. Verify the open main menu goes over the card on the banner and not under it.

# Affected Projects or Products
- D8CORE
- Note this was removed from Decanter as part of a pull request there. So that branch needs to be deleted.

# Associated Issues and/or People
- D8CORE-1396
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
